UPGRADE-6.8.md

```diff
@@ -1,27 +1,37 @@
 # 6.8.0.0
-## Introduced in 6.7.3.0
-## Removal of deprecated controller variables
-The following will be removed in Shopware 6.8.0:
-* Twig variables `controllerName` and `controllerAction`
-* CSS classes `is-ctl-*` and `is-act-*`
-* JavaScript window properties `window.controllerName` and `window.actionName`
-## Removal of "sw-empty-state"
-* The old `sw-empty-state` component will be removed in the next major version. Please use the new `mt-empty-state` component instead.
 
-Before:
-```html
-<sw-empty-state title="short title" subline="longer subline" />
-```
-After:
-```html
-<mt-empty-state title="short title" description="longer description"/>
-```
-## Removal of properties in `ResolveRemoteThumbnailUrlExtension`
+# Changed Functionality
 
-The properties `$mediaPath` and `$mediaUpdatedAt` from `Shopware\Core\Content\Media\Extension\ResolveRemoteThumbnailUrlExtension` were removed. Set the values directly into the `mediaEntity` property.
-## Removal of `hasChildren` variable 
+<details>
+
+## Tax Calculation for percentage discounts / surcharges, e.g. promotions
+
+Taxes of percentage prices are not recalculated anymore, but use the existing tax calculation of the referenced line items.
+This prevents rounding errors when calculating taxes for percentage prices.
+
+## Payment: Removal of Payment Method "Debit Payment"
+
+The payment method `DebitPayment` has been removed as it did not fulfill its purpose.
+If the payment method is and was not used, it will be removed.
+Otherwise, the payment method will be disabled.
+
+## Use orders primary delivery and primary transaction
+
+For user interfaces that display only one delivery & transaction, there is now a new reference in the order for a `primaryOrderDelivery` or `primaryOrderTransaction`.
+If an extension modifies or adds new deliveries or transactions, this should be taken into account.
+To partly comply with old behaviour, primary deliveries are ordered first and primary transactions are ordered last wherever appropriate.
+
+* Replace delivery accesses like `order.deliveries.first()` or `order.deliveries[0]` with `order.primaryOrderDelivery`
+* Replace transaction accesses like `order.transactions.last()` or `order.transactions[length - 1]` with `order.primaryOrderDelivery`
```

Comment:
Does this really fit under `Changed Functionality`? Sounds to me like a `Core` change on how you use the order entity 🤔